Formula/log4cplus.rb

```diff
@@ -11,7 +11,10 @@ class Log4cplus < Formula
     sha256 "8819df40a1477a3fb87a4a3dcd36ec8e2292b5aed362851549e8e2df74ffad9d" => :el_capitan
   end
 
+  option "with-cxx11", "set -std=c++11 in the envvar CXXFLAG"
```

Comment:
Let's always build with C++11? Using `needs :cxx11` here…

Reply:
sure, works for me